src/components/ItemList.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+const ItemList = ({ isOpen, onClose, items, onAddItem }) => {
+  if (!isOpen) return null;
+
+  return (
+    <div className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-50">
+      <div className="relative w-3/4 p-4 bg-white rounded shadow-lg">
+        <button
+          onClick={onClose}
+          className="absolute p-2 text-white bg-red-500 rounded-full top-2 right-2"
+        >
+          X
+        </button>
```

Comment:
Do buttona dodaj proszę atrybut `aria-label` który będzie przydatny dla czytników ekranowych (accessibility) na przyklad "Close button" + dodaj span który opakuje "X" i dodaj mu aria-hidden="true" aby ukryć tą literkę z accessibility tree, wtedy czytniki ekranowe nie odcztają userowi słowa "X" tylko to co będzie w aria-label